napalm_logs/base.py

```diff
@@ -56,34 +48,30 @@ def __init__(self,
                  extension_config_path=None,
                  extension_config_dict=None,
                  log_level='warning',
-                 log_format='%(asctime)s,%(msecs)03.0f [%(name)-17s][%(levelname)-8s] %(message)s'):
+                 log_format='%(asctime)s,%(msecs)03.0f [%(name)-17s][%(levelname)-8s] %(message)s',
+                 log_syslog=None,
+                 auth={},
+                 listener={},
+                 logger={},
```

Comment:
The logger and other not critical to the application can be like that, but not those that are very important.